examples/11-mobile-demo/plugin.html

```diff
@@ -72,7 +72,8 @@ <h2>This plugin is:</h2>
     <div class="calendar-backdrop"></div>
 
     <script>
-        import $ from '@windy/$';
+        import _ from '@windy/utils';
+        const {$} = _;
```

Comment:
Please, use prettier to keep formatting consistent. Also, imports first, declarations below.